### PR TITLE
Fix BitSnoop

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Fix https://github.com/pymedusa/Medusa/issues/1410#issuecomment-269413532

 @OmgImAlexis

### CODE REVIEW

**Comment by medariox** [comment]
The issue is that row.enclosure is None, this won't fix it.

**Comment by duramato** [comment]
won't the .get return none? there's a check for download_url right ahead of this

**Comment by medariox** [comment]
If you try .get on a None, you will still get an exception. get is useful if you have e.g. dict, but you don't know if that dict has a specific key. I removed the enclosure now, as it had a toracache link anyway. Let me know what you think.

File: medusa/providers/torrent/xml/bitsnoop.py [hunk 1]
```diff
@@ -101,29 +101,28 @@ def parse(self, data, mode):
...
+                return items
+
+            for element in elements:
                 try:
-                    if not row.category.text.endswith(('TV', 'Anime')):
+                    if not element.category.text.endswith(('TV', 'Anime')):
                         continue
 
-                    title = row.title.text
-                    # Use the torcache link bitsnoop provides,
```
Comment (duramato) [inline medusa/providers/torrent/xml/bitsnoop.py]: This should stay

Comment (medariox) [inline medusa/providers/torrent/xml/bitsnoop.py]: What should stay and why?

Comment (duramato) [inline medusa/providers/torrent/xml/bitsnoop.py]: Ignore me lol, all good here

**Comment by duramato** [comment]
LGTM

**Pull request merged**